OpenSim/Examples/Plugins/VolumetricContactExample/VolumetricContactForce.h

```diff
@@ -0,0 +1,134 @@
+#ifndef _OPENSIM_VOLUMETRIC_CONTACT_FORCE_PLUGIN_H_
+#define _OPENSIM_VOLUMETRIC_CONTACT_FORCE_PLUGIN_H_
+/* -------------------------------------------------------------------------- *
+ *                        VolumetricContactForce.h                            *
+ * -------------------------------------------------------------------------- *
+ * The OpenSim API is a toolkit for musculoskeletal modeling and simulation.  *
+ * See http://opensim.stanford.edu and the NOTICE file for more information.  *
+ * OpenSim is developed at Stanford University and supported by the US        *
+ * National Institutes of Health (U54 GM072970, R24 HD065690) and by DARPA    *
+ * through the Warrior Web program.                                           *
+ *                                                                            *
+ * Copyright (c) 2005-2017 Stanford University and the Authors                *
+ * Author(s):                                                                 *
+ *                                                                            *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may    *
+ * not use this file except in compliance with the License. You may obtain a  *
+ * copy of the License at http://www.apache.org/licenses/LICENSE-2.0.         *
+ *                                                                            *
+ * Unless required by applicable law or agreed to in writing, software        *
+ * distributed under the License is distributed on an "AS IS" BASIS,          *
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied    *
+ * See the License for the specific language governing permissions and        *
+ * limitations under the License.                                             *
+ * -------------------------------------------------------------------------- */
+
+//=============================================================================
+// INCLUDES
+//=============================================================================
+// Headers define the various property types that OpenSim objects can read 
+#include <string>
+#include "osimPluginDLL.h"
+#include <OpenSim/Simulation/Model/Force.h>
+#include <OpenSim/Simulation/Model/TwoFrameLinker.h>
+
+
+//=============================================================================
+//=============================================================================
+/*
+ * A class template for writing a custom Force plugin. 
+ * Applies ...
+ * @author         
```

Comment:
Don't be shy to list yourself as the author. Refer to other works where appropriate.

Reply:
Where is it appropriate to list other works? Certainly it would be useful to, but I do not know how visible most of these are to users. What documentation do most users look at? The source code? the C++ API?